components/server/src/workspace/workspace-starter.ts

```diff
@@ -62,6 +62,22 @@ export class WorkspaceStarter {
                 await this.workspaceDb.trace({ span }).store(workspace);
             }
 
+            if (forceDefault) {
+                const req = new ResolveBaseImageRequest();
+                req.setRef("gitpod/workspace-full");
```

Comment:
Could we use here the workspace image configured in the `values.yaml`? https://github.com/gitpod-io/gitpod/blob/3b1b7f11183c4972bd08478a3e44b183edf5ada1/chart/values.yaml#L255-L258

See e.g. here: https://github.com/gitpod-io/gitpod/blob/3b1b7f11183c4972bd08478a3e44b183edf5ada1/components/server/src/workspace/config-provider.ts#L284